scalableminds/webknossos
# [WIP] Status bar

This PR adds a status bar to a tracing view to display important information and shortcuts.

### URL of deployed dev instance (used for testing):
- https://statusbar.webknossos.xyz

### Steps to test:
- open tracing
- there should be a status bar displaying the current mag 
- when moving the mouse in a (non 3d view) plane the position and cell-id should be displayed
- there should be shortcuts for moving, moving along 3rd axis and zooming
- in the 3d viewport an additional shortcut for rotating the 3d view should be displayed 

### Issues:
- fixes #4995

------
